### PR TITLE
docs: move feature list into table

### PR DESCRIPTION
**Pull request by zhenjunMa**
**What this PR does**:

Modify the description of various APIs in READ.ME into a table format to improve the reading experience.

### CODE REVIEW

**Review by seeflood — CHANGES_REQUESTED** [review]
I just modified the description in your branch.
Please add File API and binding API related docs

**Review by seeflood — CHANGES_REQUESTED** [review]
Also,please add docs for using wasm for sidecar extension (not as faas)

**Comment by zhenjunMa** [comment]
@seeflood  fixed

File: docs/en/start/wasm/start.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+## WASM on Layotto
```
Comment (seeflood) [inline docs/en/start/wasm/start.md:1]: please add this doc into sidebar

File: docs/zh/start/wasm/start.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+## WASM概述
```
Comment (seeflood) [inline docs/zh/start/wasm/start.md:1]: please add this doc into sidebar

File: docs/zh/README.md [hunk 3]
```diff
@@ -39,49 +39,63 @@ Layotto 提供了多种语言版本的 SDK，SDK 通过 gRPC 与 Layotto 进行
...
 
-[作为 istio 的数据面](zh/start/istio/start.md)
+|  feature       | status |                       quick start                      |                               desc                         |
+|  ------------- | :----: | :-----------------------------------------------------:|------------------------------------------------------------|
+| Go (TinyGo)    | ✅     | [demo](https://mosn.io/layotto/#/zh/start/wasm/start)  | 把用 TinyGo 开发的代码编译成 *.wasm文件跑在 Layotto 上         |
+| Rust           | 待开发     |  待开发 | 把用 Rust 开发的代码编译成 *.wasm文件跑在 Layotto 上           |
+| AssemblyScript | 待开发     |  待开发 | 把用 AssemblyScript 开发的代码编译成 *.wasm文件跑在 Layotto 上 |
 
-### 健康检查、查询运行时元数据
+### 通过WebAssembly (WASM)做扩展、写FaaS
```
Comment (seeflood) [inline docs/zh/README.md:86]: 这两段的标题都是 “通过WebAssembly (WASM)做扩展、写FaaS”
需要区分下

**Review by seeflood — CHANGES_REQUESTED** [review]

**Review by seeflood — APPROVED** [review]
LGTM

**Pull request merged**